go/libkb/loaduser.go

```diff
@@ -438,6 +438,6 @@ func lookupMerkleLeaf(ctx context.Context, g *GlobalContext, uid keybase1.UID, l
 	return
 }
 
-func LoadUserPlusKeys(ctx context.Context, g *GlobalContext, uid keybase1.UID) (keybase1.UserPlusKeys, error) {
-	return g.GetUPAKLoader().LoadUserPlusKeys(ctx, uid)
+func LoadUserPlusKeys(ctx context.Context, g *GlobalContext, uid keybase1.UID, kid keybase1.KID) (keybase1.UserPlusKeys, error) {
```

Comment:
While the protocol field is pollForKID, it's unclear looking at this kid field what it is for.  Could you add a comment and/or change the name?